draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1131,6 +1131,11 @@ explicitly. Horizontal tab (%x09) is excluded from the CTL characters that
 lead to set-cookie-string rejection, as it is considered whitespace, which is
 handled separately.
 
+NOTE: The set-cookie-string may contain octet sequences that appear
+percent-encoded as per {{Section 2.1 of RFC3986}}. However, a user agent
```

Comment:
This should be added to the informative references at the top.

Reply:
I think it's already there, but maybe I'm missing something.

Reply:
Nope, I skimmed too quickly. My bad!